wikia/common/mw_database/connection.py

```diff
@@ -28,10 +42,27 @@ def _query(self, query, *args, **kwargs):
             log_text += ' (with args: {})'.format(kwargs['args'])
         logger.debug(log_text)
         def do_exec_query(cursor):
+            time_started = time.time()
+
             returned = cursor.execute(query, *args, **kwargs)
             affected = cursor.rowcount
             all_rows = cursor.fetchall()
 
+            # log SQL - sampled at 1%
+            if random.random() < 0.01:
+                extra = {}
+                extra.update(self.connection_info.__dict__ or {})
+                extra.pop('password', None)
+                extra['affected'] = affected
```

Comment:
I have trouble catching it from the app.
Currently there is just a message: `SQL select * from page`
and a few extra fields: `username`, `hostname`, `dbname`, `cluster`, `master`, `num_rows`, `elapsed`, `script`.